docs/reference/deprecated.md

```diff
@@ -7,6 +7,7 @@ Gutenberg's deprecation policy is intended to support backwards-compatibility fo
 - `wp.editor.getColorName` has been removed. Please use `wp.editor.getColorObjectByColorValue` instead.
 - `wp.editor.getColorClass` has been renamed. Please use `wp.editor.getColorClassName` instead.
 - `value` property in color objects passed by `wp.editor.withColors` has been removed. Please use color property instead.
+- The Subheading block has been removed. Please use the Paragraph block instead.
```

Comment:
I think deprecating blocks is special and we shouldn't remove it in 3.9.0. We may remove it later when we merge to Core or when we build a deprecated versions across blocks. So I'd suggested removing this.

Reply:
Notably, the same kind of message was put in the 3.5.0 section for the Text Columns block, which is why I put this message here. I will remove my message and create a separate PR for removing the one that was created for the Text Columns block.

Reply:
Oh maybe let's keep for now then for the sake of consistency. It could be argued that it "encourages" people to migrate.

Reply:
Wow, that was fast! I actually pushed a commit to remove the message, but it looks like you merged the branch before I did that, so I guess I will just delete the branch now. :stuck_out_tongue: